serde_derive/src/internals/case.rs

```diff
@@ -6,6 +6,9 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
+//! Stuff to handle where the case of the source (e.g. `my-field`) is different to the
```

Comment:
Please change "different to" to "different from", which also works in American English.
https://en.oxforddictionaries.com/usage/different-from-than-or-to

Reply:
TIL

Reply:
I've actually reworded this completely.